### PR TITLE
Partial Ivy; Added theme switcher; Package updates

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,65 +8,55 @@
...
     "@angular-eslint/eslint-plugin-template": "12.3.1",
     "@angular-eslint/schematics": "12.3.1",
     "@angular-eslint/template-parser": "12.3.1",
     "@angular/cli": "~12.2.2",
-    "@angular/compiler-cli": "~12.2.0",
-    "@skyux-sdk/documentation-schematics": "^5.0.0-beta.1",
-    "@skyux-sdk/e2e": "^4.0.3",
+    "@angular/compiler-cli": "~12.2.5",
+    "@skyux-sdk/documentation-schematics": "^5.0.0-beta.4",
+    "@skyux-sdk/e2e": "^4.1.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json:59]: Got a beta version of e2e now.

File: projects/tiles/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "dragula": "3.7.3",
-    "ng2-dragula": "1.5.0",
+    "ng2-dragula": "2.1.1",
```
Comment (Blackbaud-SteveBrush) [inline projects/tiles/package.json:14]: Any breaking changes here?

Comment (Blackbaud-AlexKingman) [reply projects/tiles/package.json]: @Blackbaud-SteveBrush this was an oversight - I had intentions to go through and update the breaking changes once the build broke....but it never broke! Kind of strange, b/c there are changes needed.
Regardless... I don't think we need to introduce all that today, so I reverted this bump.

Ready for you again.

File: package.json (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: Updated.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**